handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -1838,7 +1838,7 @@ protected ByteBuf removeEmptyValue() {
             return null;
         }
 
-        private static boolean attemptCopyToCumulation(ByteBuf cumulation, ByteBuf next, int wrapDataSize) {
```

Comment:
nit: this could be kept static and moved outside the scope of this class.

Reply:
sure why not...